backends/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+file(GLOB_RECURSE ref_backend_sources ${PROJECT_SOURCE_DIR}/backends/ref/*)
+target_sources(CEED PUBLIC ${ref_backend_sources})
+
+if (CEED_ENABLE_AVX)
+  file(GLOB_RECURSE avx_backend_sources ${PROJECT_SOURCE_DIR}/backends/avx/*)
+  target_sources(CEED PUBLIC ${avx_backend_sources})
+endif()
+
+if (CEED_ENABLE_BLOCKED)
```

Comment:
The ref, blocked, and opt backends should always be built, as they have no external dependencies